code/modules/mob/new_player/new_player.dm

```diff
@@ -150,6 +168,22 @@
 			to_chat(usr, "\red The round is either not ready, or has already finished...")
 			return
 
+		// Warn the player if they are trying to spawn without a brain
+		var/datum/body_modification/mod = client.prefs.get_modification(BP_BRAIN)
+		if(istype(mod, /datum/body_modification/limb/amputation))
+			if(alert(src,"Are you sure you wish to spawn without a brain? This will likely cause you do die immediately. \
+			              If not, go into the Augmentation section of Setup Character and check you do not have Removed for the brain category.", \
+						  "Player Setup", "Yes", "No") == "No")
+				return 0
+
+		// Warn the player if they are trying to spawn without eyes
+		mod = client.prefs.get_modification(BP_EYES)
+		if(istype(mod, /datum/body_modification/limb/amputation))
+			if(alert(src,"Are you sure you wish to spawn without eyes? It will likely be difficult to see without them. \
+			              If not, go into the Augmentation section of Setup Character and check you do not have Removed for the eyes category.", \
```

Comment:
```suggestion
			              If not, go into the Augmentation section of Setup Character and check that you do not have Removed for the eyes category.", \
```

Reply:
Engrish and not enough idiot proofing

should be: 

"If not, go to the **Augmentation** section of **Setup Character** and change the "eyes" slot from _Removed_ to the desired kind of eyes."